academic-calendar/react/__tests__/ProgramExplorator.test.js

```diff
@@ -0,0 +1,83 @@
+import "@testing-library/jest-dom";
+import { render, fireEvent, cleanup } from "@testing-library/react";
+import { screen } from "@testing-library/dom";
+import ProgramExplorator from "../src/components/ProgramExplorator";
+import ProgramSet from "../src/components/ProgramSet";
+
+//set up a mock API call
+jest.mock("../src/api", () => jest.fn());
```

Comment:
Why is there a mocked API but no calls or references to it in the tests? The component being tested has no direct API calls.

Reply:
removed in the latest commit. Thanks again for pointing it out, Mike.